content/pages/03-people.rst

```diff
@@ -78,8 +78,8 @@ Masters Students
 PhD Students
 ------------
 
-- `Rebecca Miko`_ |br|
-  Research Interests: computational neuroscience, olfaction
+- `Rebecca Miko <https://uk.linkedin.com/in/rebecca-miko>`__ |br|
+  Research Interests: computational neuroscience, robotics and olfaction
 
 .. _Rebecca Miko: #
```

Comment:
This half of the named link needs to be removed since you've used an anonymous link there instead.